Created and Updated a Python Audiobook

ishitaojha opened the pull request:
Alexa, Jarvis and Siri work on the same mechanism with minor changes. 
People will definitely find this helpful. Please merge it :) 😇

ishitaojha committed:
Created and Updated a Python Audiobook

Alexa, Jarvis and Siri work on the same mechanism with minor changes. 
People will definitely find this helpful. Please merge it :) 😇

srinidh-007 reviewed (commented):
Write Documentation/Comments for explaining your code

ishitaojha commented:
Sure , I was a bit busy because of my college, Will update it soon.

ishitaojha committed:
updated commands

Hey, it's my first time contributing so I don't know how to add documentation but I have incorporated some commands and comments hope that help. If you don't like it then no issues . Just don't mark it spam. I will learn more about Documentation 😊😊

ishitaojha commented:
I tried to edit the changes but somehow they didn't got merged ,
as it was my first time please oblige me 😊 
I will learn more about adding documentation 📌
PS : Following you because of the amazing repositories 
Thank you for the experience 👍🤩
